packages/coinstac-ui/test/e2e/test.js

```diff
@@ -0,0 +1,140 @@
+const Application = require('spectron').Application;
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const path = require('path');
+
+const electronPath = path.join(__dirname, '../..', 'node_modules', '.bin', 'electron');
+const appPath = path.join(__dirname, '../..');
+const mocksPath = path.join(__dirname, 'mocks.js');
+
+const EXIST_TIMEOUT = 4000;
+const USER_ID = 'test';
+const PASS = 'test';
+const CONS_NAME = 'e2e-consortium';
+const CONS_DESC = 'e2e-description';
+const PROJECT_NAME = 'e2e-files';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const app = new Application({
+  path: electronPath,
+  env: { NODE_ENV: 'test' },
+  args: [appPath, '-r', mocksPath],
+});
+
+describe('Testing::e2e', () => {
+  before(() => {
```

Comment:
You can drop the `return` and use the arrow function’s implicit return:

```diff
-  before(() => {
-    return app.start();
-  });
+ before(() => app.start());
```

Not sure why the linter isn’t prompting you to do this (probably need to upgrade).